newrelic/api/log.py

```diff
@@ -69,11 +55,46 @@ def safe_json_encode(obj, ignore_string_types=False, **kwargs):
 class NewRelicContextFormatter(logging.Formatter):
     DEFAULT_LOG_RECORD_KEYS = frozenset(set(vars(logging.LogRecord("", 0, "", 0, "", (), None))) | {"message"})
 
-    def __init__(self, *args, **kwargs):
-        super(NewRelicContextFormatter, self).__init__()
+    def __init__(self, *args, stack_trace_limit: Optional[int] = 0, **kwargs):
```

Comment:
```suggestion
    def __init__(self, *args, stack_trace_limit=0, **kwargs):
```

We still have Python 2 wheels at the moment, so this line is a syntax issue. Should be removed.

Reply:
Actually having a default keyword argument after *args isn't supported in Python 2 at all, I'll remove it for now like we do for our trace APIs and just attempt to pop it off the kwargs list if it exists.